src/jemalloc.c

```diff
@@ -1447,10 +1447,15 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 				CONF_CONTINUE;
 			}
 
-			CONF_HANDLE_SIZE_T(
+			CONF_HANDLE_UINT64_T(
 			    opt_hpa_opts.hugify_delay_ms, "hpa_hugify_delay_ms",
 			    0, 0, CONF_CHECK_MIN, CONF_DONT_CHECK_MAX, true);
 
+			CONF_HANDLE_UINT64_T(
+			    opt_hpa_opts.min_purge_interval_ms,
+			    "hpa_min_purge_interval_ms",
+			    0, 0, CONF_CHECK_MIN, CONF_DONT_CHECK_MAX, true);
```

Comment:
With the uint64 type, `CONF_CHECK_MIN` is not needed. Also the `clip` (last arg) should be false since it doesn't really have a range.

Reply:
Makes sense. I basically reused these parameters from `hpa_hugify_delay_ms` above, which should also be corrected accordingly.